http/http.go

```diff
@@ -234,6 +234,8 @@ func setBody(req *http.Request, body starlark.String, formData *starlark.Dict, f
 			return err
 		}
 		req.Body = ioutil.NopCloser(strings.NewReader(uq))
+		req.ContentLength = int64(len(uq))
```

Comment:
Can you add a comment here clarifying the need to explicitly set this value? The rationale outlined in your PR would suffice.

Reply:
Hello @b5 - all set and comment added.   Let me know if ok/or anything else needed.   Have a great weekend!

Reply:
Oops - and forgot to mention. Compiled clean and tested.   Take care.